### ISSUE TITLE
@ApplicationModuleTest does not work in subclass

### ISSUE THREAD

**Issue by michaldo** [spring-projects-experimental/spring-modulith#176]
Hi, example:
```
// @SpringBootTest  -- it works
@ApplicationModuleTest abstract class MyBaseTest {}

class MyConcreteTest extends MyBaseTest {

    @Test
    void contextLoads() {
    }
}
```
> Starting MyConcreteTest 
> A component required a bean of type 'org.springframework.modulith.test.ModuleTestExecution' that could not be found.

**Comment by odrotbohm** [comment]
This sounds like a duplicate of #173 . Would you mind trying the 0.6 snapshots?

**Comment by midovlvn** [comment]
Fixed in 0.6.0-SNAPSHOT

**Comment by odrotbohm** [comment]
Awesome, happy coding! 🙃
